MMM-COVID19.js

```diff
@@ -130,47 +143,61 @@ Module.register("MMM-COVID19", {
           newCasesCell = document.createElement("td"),
           deathsCell = document.createElement("td"),
           newDeathsCell = document.createElement("td"),
+          seriousCell = document.createElement("td"),
           recoveredCell = document.createElement("td"),
           activeCell = document.createElement("td"),
+          casesPerMCell = document.createElement("td"),
           cases = globalStats["total_cases"],
           newCases = globalStats["new_cases"],
           deaths = globalStats["total_deaths"],
           newDeaths = globalStats["new_deaths"],
           totalRecovered = globalStats["total_recovered"],
-          activeCases = '';
+          activeCases = this.translate('N/A'),
+          serious = this.translate('N/A'),
+          casesPerM = this.translate('N/A');
 
       worldNameCell.innerHTML = this.translate('Worldwide')
       worldNameCell.className = this.config.infoRowClass
       worldRow.className = 'world ' + this.config.infoRowClass
       confirmedCell.className = 'number confirmed ' + this.config.infoRowClass
       confirmedCell.innerHTML = cases
+      casesPerMCell.className = 'number cases per m ' + this.config.infoRowClass
+      casesPerMCell.innerHTML = casesPerM
       newCasesCell.className = 'number confirmed ' + this.config.infoRowClass
       if (newCases) {
-        newCasesCell.innerHTML = newCases
+        newCasesCell.innerHTML = '+' + newCases
       }
       deathsCell.className = 'number deaths ' + this.config.infoRowClass
       deathsCell.innerHTML = deaths
       newDeathsCell.className = 'number deaths ' + this.config.infoRowClass
       if (newDeaths) {
-        newDeathsCell.innerHTML = newDeaths
+        newDeathsCell.innerHTML = '+' + newDeaths
       }
+      seriousCell.className = 'number deaths ' + this.config.infoRowClass
```

Comment:
@testdruif wrong className used here. Don't use the same color as Deaths. If you want to highlight it someway, please add a new class called `serious` and add the proper entry at the .css file of the module